doc/programming_guide/event_loop.rst

```diff
@@ -5,34 +5,154 @@ Introduction
+Firstly it inforces that the ``delta_time`` passed into its ``tick`` method is always the same.
+This is because advanced physics engines require consistent time. Secondly the fixed clock
+requires a sibling regular clock. It uses this clock to track how offset from the true time it is.
+Like the regular clock you may make a new :py:class:`arcade.FixedClock` at any time,
+but ensure they have a sibling.
+
+Up Coming
+^^^^^^^^^
+In future versions of arcade there will be a :py:class:`arcade.SubClock`. The sub clock will
+provide many useful features currently missing from regular and fixed clocks. Sub clocks are 
+ticked by their parent clock. This means you won't need to manually tick clocks created at runtime.
+Sub clocks will also allow for time manipulation by changing how fast they see time flow. This
+allows you to easily create slow-down effects without any advanced maths or changing the number of 
+updates per frame. To gain access to a draft :py:class:`arcade.SubClock` you can find it in
+``arcade.future.sub_clock``. This version of the sub clock is not final. If you find any bugs do
+not hesitate to raise an issue on the github.
+
+More on Fixed update
+--------------------
+the ``on_fixed_update`` event can be an extremelly powerful tool, but it has many complications
+that should be accounted for. If used imporperly the event can grind a game to a halt.
+
+Death Spiral
+^^^^^^^^^^^^
+A fixed update represnts a very specific amount of time. If all of the computations take
+longer than the fixed update represents than the ammount of time accumulated between update
+events will grow. If this happens for multiple frames the game will begin to spiral. The 
+first few frames of the spiral will lead to one update cycle requiring two fixed update
+calls. This will increase the extra time accumulated until three fixed updates must occur at once.
+This will continue to happen until either: the fixed updates start taking less time, or the game 
+crashes.
+
+There are a few solutions to this issue. The simplist method, which works best when there may be spikes in
+computation time that quickly settle, is to clamp the max number of fixed updates that can occur in a single
+frame. In arcade this is done by setting the ``fixed_frame_cap`` argument when initialising your
+:py:class:`arcade.Window`. The second method is to slow-down time temporarily. By changing the
+``_tick_speed`` of arcade's ``GLOBAL_CLOCK`` is is possible to slow down the accumulation of time.
+For example setting ``GLOBAL_CLOCK._tick_speed = 0.5`` would allow the fixed update twice as many frames
+to calculate for.
+
+Update Interpolation
+^^^^^^^^^^^^^^^^^^^^
+Because fixed updates work on the accumulation of time this may not sync with the perfectly with 
+the ``on_draw`` or ``on_update`` events. In extreme cases this can cause a visible stuttering to
+objects moved within ``on_fixed_update``. This is where the ``accumulated`` and ``fraction`` properties
```

Comment:
To prevent this, ``GLOBAL_FIXED_CLOCK`` provides ``accumulated`` and ``fraction`` properties.